packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -471,6 +474,11 @@ class ListView extends BoxScrollView {
   ///
   /// [itemBuilder] will be called only with indices greater than or equal to
   /// zero and less than [itemCount].
+  ///
+  /// The builder should actually create the widget instances when called. Avoid
+  /// using a builder that returns previously-constructed list of widgets; it is
```

Comment:
using an item builder that returns a previously constructed widget. If the list view's children are going to be created in advance it's more efficient to use [new ListView].